23/08/2023 10:14:31: Error in register_atlas.py for sub-CC00056XX07: Command 'python /mnt/c/Users/ArcaroLab/Desktop/git_repos/dhcp/fmri/register_atlas.py sub-CC00056XX07 ses-10700 object' returned non-zero exit status 1.
23/08/2023 10:14:32: Error in register_atlas.py for sub-CC00057XX08: Command 'python /mnt/c/Users/ArcaroLab/Desktop/git_repos/dhcp/fmri/register_atlas.py sub-CC00057XX08 ses-11002 object' returned non-zero exit status 1.
23/08/2023 10:14:34: Error in register_atlas.py for sub-CC00058XX09: Command 'python /mnt/c/Users/ArcaroLab/Desktop/git_repos/dhcp/fmri/register_atlas.py sub-CC00058XX09 ses-11300 object' returned non-zero exit status 1.
23/08/2023 10:19:49: Error in register_atlas.py for sub-CC00056XX07: Command 'python /mnt/c/Users/ArcaroLab/Desktop/git_repos/dhcp/fmri/register_atlas.py sub-CC00056XX07 ses-10700 object' returned non-zero exit status 1.
23/08/2023 10:25:05: Error in register_atlas.py for sub-CC00056XX07: Command 'python /mnt/c/Users/ArcaroLab/Desktop/git_repos/dhcp/fmri/register_atlas.py sub-CC00056XX07 ses-10700 object' returned non-zero exit status 1.
23/08/2023 10:25:40: Error in register_atlas.py for sub-CC00056XX07: Command 'python /mnt/c/Users/ArcaroLab/Desktop/git_repos/dhcp/fmri/register_atlas.py sub-CC00056XX07 ses-10700 object' returned non-zero exit status 1.
23/08/2023 10:25:41: Error in register_atlas.py for sub-CC00057XX08: Command 'python /mnt/c/Users/ArcaroLab/Desktop/git_repos/dhcp/fmri/register_atlas.py sub-CC00057XX08 ses-11002 object' returned non-zero exit status 1.
23/08/2023 11:46:04: Error in register_atlas.py for sub-CC00056XX07: Command 'python /mnt/c/Users/ArcaroLab/Desktop/git_repos/dhcp/fmri/register_atlas.py sub-CC00056XX07 ses-10700 object' returned non-zero exit status 1.
23/08/2023 11:46:06: Error in register_atlas.py for sub-CC00057XX08: Command 'python /mnt/c/Users/ArcaroLab/Desktop/git_repos/dhcp/fmri/register_atlas.py sub-CC00057XX08 ses-11002 object' returned non-zero exit status 1.
